scan/src/pages/ValidatorHomePage.re

```diff
@@ -94,75 +94,33 @@ let renderBody = (rank, validator: ValidatorSub.t, bondedTokenCount) => {
   // let reportRate = (validator.completedRequestCount |> float_of_int) /. allRequestCount *. 100.;
 
   <TBody key={rank |> string_of_int}>
```

Comment:
Shouldn't key be the validator address? Don't think rank is the right value for key?